tests/methods/avas-8/input.dat

```diff
@@ -0,0 +1,114 @@
+# Test the pi orbital selection of AOSubspace on overlapped planes C20 molecule
+
+import forte
+
+refscf = -756.50604655444420
+reffci = -113.911667467206257
+
+molecule C20{
+# grabed from https://nanotube.msu.edu/fullerene/fullerene.php?C=20
+# C1 symmetry due to Jahn-Teller distortion
+C        1.5691000000     -0.6566000000     -0.9364000000
+C        1.7669000000      0.6431000000     -0.4720000000
+C        0.4705000000     -0.6652000000     -1.7927000000
+C        0.0116000000      0.6478000000     -1.8255000000
+C        0.7930000000      1.4673000000     -1.0284000000
+C       -0.4874000000     -1.4818000000     -1.2157000000
+C       -1.5635000000     -0.6572000000     -0.8952000000
+C       -1.2694000000      0.6490000000     -1.2767000000
+C       -0.0023000000     -1.9618000000     -0.0072000000
+C       -0.7698000000     -1.4532000000      1.0359000000
+C       -1.7576000000     -0.6380000000      0.4742000000
+C        1.2878000000     -1.4503000000      0.1629000000
+C        1.2896000000     -0.6595000000      1.3047000000
+C        0.0115000000     -0.6460000000      1.8533000000
+C        1.5830000000      0.6454000000      0.8984000000
+C        0.4848000000      1.4383000000      1.1937000000
+C       -0.5032000000      0.6469000000      1.7753000000
+C       -1.6062000000      0.6715000000      0.9231000000
+C       -1.2959000000      1.4891000000     -0.1655000000
+C       -0.0102000000      1.9727000000     -0.0063000000
+}
+
+set {
+  basis         cc-pvdz
+  df_basis_scf  cc-pvdz-jkfit
+  reference     rhf
+  scf_type      df
+  d_convergence 10
+}
+Escf, wfn = energy('scf', return_wfn=True)
+compare_values(refscf, variable("CURRENT ENERGY"), 8, "SCF energy")
+
+# figure out planes
+import numpy as np
+from collections import defaultdict
+from itertools import combinations
+
```

Comment:
First we grab the geometry of fullerene in cartesian coordinates and import it into the python numpy  module